### ISSUE TITLE
A way to test schedules without running them

### ISSUE THREAD

**Issue by totalhack** [gjcarneiro/yacron#80]
### Description

Apologies if I've just missed this, but is there a way to dry run schedules without actually executing the tasks? I have a number of schedules that are run in production and would like to be able to test configuration adjustments locally without actually executing tasks and without having to comment a bunch of stuff out. Ideally it would be a CLI or env var flag to specify this mode, and yacron would just print the tasks it would run.

Thanks.

**Comment by gjcarneiro** [comment]
No.  The only thing you have right now is a `--validate-config` option, that checks that the configuration file is syntactically valid.  But there's no dry run mode in yacron.

**Comment by andreas-wittig** [comment]
Couple of days ago I had a similiar question to [gjcarneiro].
My issue could be solved with ```enabled=[true, false]``` ; i.e. whether a schedule is active or not.
[totalhack]s issue is not the same, but "sounds similiar".  
A possible extension could be ```enabled=[true, false, test]``` => with ```test``` the desired magic could be done.

**Comment by totalhack** [comment]
@andreas-wittig adjusting your config that way doesn't scale well during development. A global CLI flag or env var setting would make more sense in this case IMO.
